docker-compose.solr.yaml

```
@@ -23,10 +23,24 @@ services:
     volumes:
       - .:/mnt/ddev_config
       - solr:/var/solr
-    command: bash -c "docker-entrypoint.sh solr zk cp file:/mnt/ddev_config/solr/security.json zk:/security.json && exec solr-foreground"
+    command: bash -c "
```

I would add a `set -eu` to make it fail on errors

```suggestion
    command: bash -c "set -eu
```